packages/react-components/src/templates/SharedResearchOutput.tsx

```diff
@@ -108,15 +111,18 @@ const SharedResearchOutput: React.FC<SharedResearchOutputProps> = ({
         </div>
         <div css={cardsStyles}>
           <SharedResearchOutputHeaderCard {...props} published={published} />
-          {((description && !isGrantDocument) || !!tags.length) && (
+          {((hasDescription && !isGrantDocument) || !!tags.length) && (
             <Card>
-              {description && !isGrantDocument && (
+              {hasDescription && !isGrantDocument && (
                 <div css={{ paddingBottom: `${12 / perRem}em` }}>
                   <Headline2 styleAsHeading={4}>Description</Headline2>
+                  <Markdown value={descriptionMD}></Markdown>
```

Comment:
It seems likely that older documents could have both MD and Text. I don't think you have considered this case.

Reply:
I have! Until we deal with migration, we display both, markdown first.

Reply:
Shouldn't we just prefer markdown if it exists?